### DISCUSSION TITLE
Dialog.Template.html and Localizer

### DISCUSSION THREAD

**Discussion by Zahar661** [serenity-is/Serenity#6562] [Help] [answered]
Hi all! 
How can I get a string from Text.cs in a dialog template?
 @Localizer.Get("Site.Home.Send")  not working(

**Comment by volkanceylan** [comment, marked as answer]
You can't, templates are static, e.g. one string per site, you need to move your template to javascript getTemplate method or embed template in the page. I think you are still using very old Serenity version, we used to have a {{text:"Something"}} type in html templates to access local texts, which was actually processed in script (removed in 6.2.0+).

**Reply by Zahar661** [reply]
Thank you! My 5.2.4 is not updated to 6.2, I can’t see the entries in the grids ((
